app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  has_many :likes
+  has_many :comments
+
+  after_save :update_posts_counter
+
+  def update_posts_counter
```

Comment:
- [ ] OPTIONAL
Following best practices `update_posts_counter` should be marked as private after implementing the `after_save` method.